eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/AssemblyCheckerProcessorTest.java

```diff
@@ -83,9 +83,11 @@ public static void tearDownClass() throws Exception {
 
     private SubSnpNoHgvs newSubSnpNoHgvs(String chromosome, long chromosomeStart, String contig, long contigStart,
                                          String referenceAllele, DbsnpVariantType variantClass) {
-        return new SubSnpNoHgvs(SS_ID, RS_ID, ALTERNATE_ALLELE, ASSEMBLY, "", "", chromosome, chromosomeStart, contig,
-                                variantClass, Orientation.FORWARD, Orientation.FORWARD, Orientation.FORWARD,
-                                contigStart, true, true, referenceAllele, null, TAXONOMY);
+        return new SubSnpNoHgvs(SS_ID, RS_ID, referenceAllele, ALTERNATE_ALLELE, ASSEMBLY, "", "",
```

Comment:
Try to write as much parameters in each line as possible (without exceed 121 characters).